upgrade/scripts/common/ncn-rebuild-common.sh

```diff
@@ -246,46 +246,7 @@ state_recorded=$(is_state_recorded "${state_name}" ${target_ncn})
 if [[ ${state_recorded} == "0" ]]; then
     echo "====> ${state_name} ..."
     {
-
-        target_ncn_mgmt_host="${target_ncn}-mgmt"
-        if [[ ${target_ncn} == "ncn-m001" ]]; then
-            target_ncn_mgmt_host=$(ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null ncn-m001 "ipmitool lan print | grep 'IP Address' | grep -v 'Source'"  | awk -F ": " '{print $2}')
-        fi
-        echo "mgmt IP/Host: ${target_ncn_mgmt_host}"
-
-        # retrieve IPMI username/password from vault
-        VAULT_TOKEN=$(kubectl get secrets cray-vault-unseal-keys -n vault -o jsonpath='{.data.vault-root}' | base64 -d)
-        # Make sure we got a vault token
-        [[ -n ${VAULT_TOKEN} ]]
-
-        # During worker upgrades, one vault pod might be offline, so we look for one that works.
-        # List names of all Running vault pods, grep for just the cray-vault-# pods, and try them in
-        # turn until one of them has the IPMI credentials.
-        IPMI_USERNAME=""
-        IPMI_PASSWORD=""
-        for VAULT_POD in $(kubectl get pods -n vault --field-selector status.phase=Running --no-headers \
-                            -o custom-columns=:.metadata.name | grep -E "^cray-vault-(0|[1-9][0-9]*)$") ; do
-            IPMI_USERNAME=$(kubectl exec -it -n vault -c vault "${VAULT_POD}" -- sh -c \
-                "export VAULT_ADDR=http://localhost:8200; export VAULT_TOKEN=`echo ${VAULT_TOKEN}`; \
-                vault kv get -format=json secret/hms-creds/${TARGET_MGMT_XNAME}" | 
-                jq -r '.data.Username')
-            # If we are not able to get the username, no need to try and get the password.
-            [[ -n ${IPMI_USERNAME} ]] || continue
-            IPMI_PASSWORD=$(kubectl exec -it -n vault -c vault "${VAULT_POD}" -- sh -c \
-                "export VAULT_ADDR=http://localhost:8200; export VAULT_TOKEN=`echo ${VAULT_TOKEN}`; \
-                vault kv get -format=json secret/hms-creds/${TARGET_MGMT_XNAME}" | 
-                jq -r '.data.Password')
-            export IPMI_PASSWORD
-            break
-        done
-        # Make sure we found a pod that worked
-        [[ -n ${IPMI_USERNAME} ]]
-
-        # power cycle node
-        ipmitool -I lanplus -U "${IPMI_USERNAME}" -E -H "${target_ncn_mgmt_host}" chassis power off
-        sleep 20
-        ipmitool -I lanplus -U ${IPMI_USERNAME} -E -H $target_ncn_mgmt_host chassis power status
-        ipmitool -I lanplus -U ${IPMI_USERNAME} -E -H $target_ncn_mgmt_host chassis power on
+        ssh "${target_ncn}" 'ipmitool chassis power cycle'
```

Comment:
This looks like it is removing for all NCNs and not must master like we discussed in stand-up. Am I reading that correctly?

Reply:
yes, this is going to remove it from storage nodes too for now but i think once @leliasen-hpe get storage upgrade moved to argo it is only going to be used by master node. worker nodes are already not using this

Reply:
Storage node upgrade moved to argo is planned for 1.3 as well?